speech_to_phrase/sentences/de.yaml

```
@@ -85,10 +90,10 @@ data:
   #     - "lock"
 
   # generic on/off
```

I think you missed "sentences" here.

```suggestion
  # generic on/off
  - sentences:
```